bindings/python/crocoddyl/multibody/contacts/contact-1d.cpp

```diff
@@ -68,7 +76,10 @@ void exposeContact1D() {
                     &ContactModel1D::set_reference, "reference contact translation")
       .add_property("gains",
                     bp::make_function(&ContactModel1D::get_gains, bp::return_value_policy<bp::return_by_value>()),
-                    "contact gains");
+                    "contact gains")
+      .add_property("pinReferenceFrame",
```

Comment:
1. Renamed this as suggested above
2. Add the getter and setter for the `mask`

Reply:
addressed in [fd58261](https://github.com/loco-3d/crocoddyl/pull/1054/commits/fd582614a34b626d8cb0751df8132d31f643845b)